docs/pages/eas/workflows/upgrade.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Upgrade to Workflows
```

Comment:
```suggestion
title: Upgrade to EAS Workflows
```

If we include the complete feature name in the page title, it'll be good for SEO.